scripts/run.py

```diff
@@ -7,10 +7,10 @@
 import pyhf
 import warnings
 import tensorflow as tf
-import wandb
 from datetime import datetime
 from pathlib import Path
 from plot import plot
+from stats import SystemStats
```

Comment:
Given that you're trying to have code import from each other it is probably a good idea to actually make this a real `pip` instalable package in the next PR so that this is made a lot cleaner.

Reply:
Okay. I think I will solve this problem in the next PR and I will resolve this conversation in this PR